packages/shopify-api/docs/guides/oauth.md

```diff
@@ -65,16 +65,18 @@ shopifyApi({
 
 ##### Shopify managed installation
 If your access scopes are [configured through the Shopify CLI](https://shopify.dev/docs/apps/tools/cli/configuration), scope changes will be handled by Shopify automatically.
-Learn more about [Shopify managed installation](https://shopify.dev/docs/apps/auth/installation#shopify-managed-installation). Using token exchange will guarantee that the access token retrieved will always have the latest access scopes.
+Learn more about [Shopify managed installation](https://shopify.dev/docs/apps/auth/installation#shopify-managed-installation).
+Using token exchange will ensure that the access token retrieved will always have the latest access scopes granted by the user.
 
 ##### Not using Shopify managed installation - not recommended
 If you don't have access scopes configured through the Shopify CLI, you can still use token exchange to exchange the current user's session token for access token.
-This is not recommended because you'll have to managed both OAuth flow.
+This is not recommended because you'll have to manage both OAuth flows.
```

Comment:
Nit: should this be an info tag?
```suggestion

> [!WARNING]
> This is not recommended because you'll have to manage both OAuth flows.
```

(as per https://github.com/orgs/community/discussions/16925)